go/vt/logutil/planetscale_logger_test.go

```diff
@@ -0,0 +1,119 @@
+/*
+Copyright 2019 The Vitess Authors.
```

Comment:
sorry about the nit-picking - it's year 2022 for another week :)

Reply:
Should I update it to **2023** now 🤣